README.md

```diff
@@ -1,16 +1,20 @@
-# Marked
+<a href="https://marked.js.org">
+  <img width="60px" height="60px" src="https://marked.js.org/img/logo-black.svg" align="right" />
+</a>
 
-Marked is 
+# Marked
 
-1. built for speed.
-2. a low-level markdown compiler for parsing markdown without caching or blocking for long periods of time.
-3. light-weight while implementing all markdown features from the supported flavors & specifications.
-4. available as a command line interface (CLI) and running in client- or server-side JavaScript projects.
+- ⚡ built for speed
+- ⬇️ low-level markdown compiler for parsing without caching or blocking for long periods of time
+- ⚖️ light-weight while implementing all markdown features from the supported flavors & specifications
+- 🌐 works in the browser, on the server, or on the command line interface (CLI)
```

Comment:
"...or through a command line interface" ??

Reply:
Here’s how I think:

- You run code in the browser
- You run code on the server 
- You run code on the command line
- You run code from the command line
- You run code through a command line

I think “from” might be best here. 
@uzitech Thoughts?

Reply:
"from" sound best to me